62.md

```diff
@@ -0,0 +1,112 @@
+  - Format: `["a", "<kind:pubkey:dtag>", "<relay hint>", "<event id>"]`
+  - Optional event ID field enables version tracking while maintaining updateable references
+  - Referenced events SHOULD be kind 30041 sections or nested kind 30040 indices
+  - Additional event kinds MAY be supported
+- MUST include an `auto-update` tag specifying update behavior:
+  - Format: `["auto-update", "<yes|ask|no>"]`
+  - Controls whether clients should automatically update to newer versions
+- For derivative works:
+  - MUST include a `p` tag identifying the original author
+  - MUST include an `E` tag referencing the original event immediately after the `p` tag
+
+### Example Index
+
+```json
+{
+    "id": "<event_id>",
+    "pubkey": "<event_originator_pubkey>",
+    "created_at": 1725087283,
+    "kind": 30040,
+    "tags": [
+        ["d", "aesop's-fables-by-aesop"],
+        ["title", "Aesop's Fables"],
+        ["author", "Aesop"],
+        ["i", "isbn:9780765382030"],
+        ["t", "fables"],
+        ["t", "classical"],
+        ["t", "literature"],
+        ["published_on", "2003-05-13"],
+        ["published_by", "public domain"],
+        ["image", "https://imageserver.com/piclink.jpg"],
+        ["summary", "Collection of selected fables from the ancient Greek philosopher, known as Aesop."],
+        ["a", "<kind:pubkey:dtag>", "<relay hint>", "<event id>"],
+        ["a", "<kind:pubkey:dtag>", "<relay hint>", "<event id>"],
+        ["auto-update", "<yes|ask|no>"],
+        ["p", "<pubkey_0>"],
+        ["A", "<original_event_id>", "<relay_url>", "<pubkey>"]
+    ],
+    "sig": "<event_signature>"
+}
+```
+
+## 30041: Publication Content
+
+Also known as sections, zettels, episodes, or chapters contain the actual content that makes up a collection.
+
+### Requirements
+
+- MUST include a `d` tag
+- MUST include a `title` tag describing the section (e.g., "Introduction", "Chapter 1", etc.)
+- The `content` field:
+  - MUST contain text meant for display to the end user
+  - MAY contain AsciiDoc markup
+  - MAY contain wikilinks (denoted by double brackets)
+
+### Example Section
+
+```json
+{
+    "id": "<event_id>",
+    "pubkey": "8ae74c618a4713f32129...",
+    "created_at": 1708083476,
+    "kind": 30041,
+    "tags": [
+        ["title", "The Farmer and The Snake"],
+        ["d", "aesop's-fables-by-aesop-the-farmer-and-the-snake"],
+        ["wikilink", "fable", "<pubkey>", "wss://thecitadel.nostr1.com", "<event id>"]
+    ],
+    "content": "== The Farmer and The Snake\nA [[fable]], by Aesop.\nONE WINTER a Farmer found a Snake stiff and frozen with cold. He had compassion on it, and taking it up, placed it in his bosom. The Snake was quickly revived by the warmth, and resuming its natural instincts, bit its benefactor, inflicting on him a mortal wound. 'Oh,' cried the Farmer with his last breath, 'I am rightly served for pitying a scoundrel.'\nThe greatest kindness will not bind the ungrateful.",
```

Comment:
Wikilinks in NIP-54 does not have a tag for reference and is different from this, is this intentional? How about using `nostr:...` links of NIP-21?

Reply:
You can use nostr: links in any text; that's just a rendering issue.

This is a wikilink from a note that isn't a wiki page, where the specific version might need to be specified and where the client doesn't necessarily handle wikis. We should maybe use something like the new a-tag, instead of an eventID.

Reply:
The wikilink specification here intentionally diverges from NIP-54.  I agree we could use a parameterized replaceable event reference like with `a` tags in the wikilink.

One of the goals with wikilinks, in terms of client implementation, is that they always "just work."  Even if, for one reason or another, the specific referenced event can't be found, a client might tie the wikilink to a search feature to find related events by keyword.

That's just one possibility, but that's why we're both diverging from NIP-54 somewhat, and not just using `nostr:` links.

Reply:
Ok I see. Thanks for explaining.